dnstap-sensor.8

```diff
@@ -86,6 +105,21 @@ of "0" turns off statistics logging.
 .B --trace
 Output additional logging to standard error for debugging.
 
+.TP
+.B --udp_output udp:\fIaddress\fB:\fIport\fB
+Send output NMSG data via UDP to \fIaddress\fR:\fIport\fR. The prefix \fBudp\fR
+may be replaced with \fBudp4\fR or \fBudp6\fR to force IPv4 of IPv6 transport.
+
+A \fB--udp_output\fR may be specified instead of or in addition to server URIs.
+If both are specified, data will be sent to both the UDP output and one of the
+configured server URIs.
+
+.Tp
+.B --mtu \fIsize\fB
+Specify the buffer size to use when sending NMSG data to \fB--udp_output\fR.
+In practice, this number should be lower than the actual interface MTU. The
+default value is 1280 for transport over Ethernet with a 1500 byte MTU.
```

Comment:
I think this part should also mention that 1280 is also the minimum. Maybe change it to `The default and minimum value...`

Reply:
The minimum is actually 512 (`nmsg.MinContainerSize`), 1280 is the default for consistency with `nmsgtool`. That said, the mtu validation code did need a fix, which has been pushed.